pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_versioning_a_product.adoc

```diff
@@ -15,15 +15,17 @@ A product must have at least one product version assigned to it.
 
 . Find the product and click the drop-down arrow>Product Details.
 
-. Enter the following information:
-  New Version:: This is the text that appears in {ProductShortName}, and the document preview.
-  URL Fragment:: This string appears in the user-facing URL for any content that belongs to this product and version. A URL fragment can include letters, numbers, hyphens, periods and underscores.
+. Provide the following information:
+  New Version:: Set the value for the new version of your product. The version displays in {ProductShortName}, and the document preview.
```

Comment:
"is displayed" here and everywhere that incorrectly says "displays"

Reply:
Thanks. Updated.